### DISCUSSION TITLE
add tranlation to another lang

### DISCUSSION THREAD

**Discussion by dodther** [zachowj/node-red-contrib-home-assistant-websocket#803] [General]
Hello. I could work on the translation into russian. What do you think about it?

**Comment by zachowj** [comment]
Thank you for your interest in helping with translations. I've slowly been moving hard-coded strings to a centralized location. I need to find the time to set up an account on something like https://lokalise.com/ so it will be easier for users to contribute. I'll try to do this in the near future.
